integration-cli/docker_cli_authz_plugin_v2_test.go

```diff
@@ -75,6 +75,36 @@ func (s *DockerAuthzV2Suite) TestAuthZPluginAllowNonVolumeRequest(c *check.C) {
 	c.Assert(assertContainerList(out, []string{id}), check.Equals, true)
 }
 
+func (s *DockerAuthzV2Suite) TestAuthZPluginDisable(c *check.C) {
```

Comment:
if you use `cli.d` here, daemon will panic and its quite obvious that `cli.d` is assigned later on. I dont think we need a comment to explain the obvious.